test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1,8 +1,13 @@
 # The following labels are used in this file:
 # - fast: short execution time, ideally less than 30 seconds
```

Comment:
If no job is using this label, then I prefer we remove them. Otherwise there will be risk that these tests will break or drift from the other configurations.

Reply:
I tend to disagree. In one test case, multiple workloads share the same config and their difference is only in the number of nodes/pods created. If the tests are written correctly, there shouldn't be a way that fast-labeled workloads will drift from the others.

Reply:
Reconsider removing the `fast` label, given that we will add `integration-test` back.

Reply:
I would keep it for now. The `fast` label is useful when trying to verify correctness of some changes locally, without running the largest, performance-labeled workloads. I also think it's better to have the workloads covered with some labels, even if we don't auto run them in any CI.